app/apps/client/admin/appWhatIsIt.html

```diff
@@ -1,15 +1,19 @@
 <template name="appWhatIsIt">
 	<section class="page-container page-home page-static content-background-color">
-		{{> header sectionName="Apps_WhatIsIt" hideHelp=true fixedHeight=true}}
+		{{#if isLoading}}
```

Comment:
The `#if` should be inside the content div, there is always content in the page.